### ISSUE TITLE
should provide a func to query workflows.

### ISSUE THREAD

**Issue by ycjcl868** [Clarifai/clarifai-javascript#26]
I want to get all workflows in current app, but the `app.workflow` not provide query.
I grab the req and res,  the curl data:

```
curl -X GET \
  https://api.clarifai.com/v2/users/{}/apps/{}/workflows \
  -H 'Cache-Control: no-cache' \
  -H 'authorization: Key {}' \
```

**Comment by kunal732** [comment]
hey @ycjcl868 we're looking into this

**Comment by kunal732** [comment]
This is coming soon, will keep you updated.

**Comment by rok-povsic** [comment]
@ycjcl868 Hi, we've released a new version (2.8.0) that contains this. Here's an example of listing all the workflows:

```js
app.workflows.list()
.then(workflows => {
    for (let i = 0; i < workflows.length; i++) {
        let workflow = workflows[i];
        console.log(i + ': ' + workflow.id + ' ' + workflow.createdAt);
    }
});
```

**Comment by ycjcl868** [comment]
@rok-povsic thanks
